charts/flagsmith/templates/deployment-api.yaml

```
@@ -80,7 +80,9 @@ spec:
         image: {{ printf "%s:%s" ( .Values.api.image.repository  ) ( .Values.api.image.tag | default .Chart.AppVersion) }}
         imagePullPolicy: {{ .Values.api.image.imagePullPolicy | default .Values.global.image.imagePullPolicy }}
         args: ["migrate"]
-        env: {{ include (print $.Template.BasePath "/_api_environment.yaml") . | nindent 8 }}
+        env: {{ include (print $.Template.BasePath "/_api_environment.yaml") . | nindent 8 -}}
+        volumeMounts:
+{{ toYaml $.Values.api.volumeMounts | indent 10 }}
```

```suggestion
{{ toYaml .Values.api.volumeMounts | indent 10 }}
```

Unless I'm missing something, I think the `$` here is unnecessary?

---

was using it to debug and missed removing it :^)

---

fixed